src/loaders/lottie/tvgLottieParser.cpp

```diff
@@ -457,12 +457,20 @@ void LottieParser::parsePropertyInternal(T& prop)
 
 
 template<typename T>
```

Comment:
Probably, we can remove a parameter, by adding: 
```
template<typename T, LottieProperty type>
void LottieParser::parseProperty(T& prop, LottieObject* obj)
```

See https://github.com/thorvg/thorvg/pull/1998